interface/modules/custom_modules/oe-module-claimrev-connect/public/claims.php

```diff
@@ -0,0 +1,334 @@
+            <?php
+               
+                include '../templates/navbar.php';
+            ?>
+            </div>
+        </div>
+        <form method="post" action="<?=$_SERVER['PHP_SELF'];?>">
+            <div class="card">  
+                <div class="row">
+                    <div class="col">
+                        <div class="form-group">
+                            <label for="startDate">Send Date Start</label>
+                            <input type="date" class="form-control"  id="startDate" name="startDate" value="<?php echo isset($_POST['startDate']) ? $_POST['startDate'] : '' ?>"  placeholder="yyyy-mm-dd"/>
+                        </div>
+                    </div>                    
+                    <div class="col">
+                        <div class="form-group">
+                            <label for="endDate">Send Date End</label>
+                            <input type="date" class="form-control"  id="endDate" name="endDate" value="<?php echo isset($_POST['endDate']) ? $_POST['endDate'] : '' ?>" placeholder="yyyy-mm-dd"/>
+                        </div>
+                    </div>
+                    <div class="col">
+                      
+                    </div>                    
+                    <div class="col">
+                        
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col">
+                        <div class="form-group">
+                            <label for="patFirstName">Patient First Name</label>
+                            <input type="text" class="form-control"  id="patFirstName" name="patFirstName"  value="<?php echo isset($_POST['patFirstName']) ? $_POST['patFirstName'] : '' ?>"  placeholder="Patient First Name"/>
+                        </div>
+                    </div>
+                    <div class="col">
+                        <div class="form-group">
+                            <label for="patLastName">Patient Last Name</label>
+                            <input type="text" class="form-control"  id="patLastName" name="patLastName"  value="<?php echo isset($_POST['patLastName']) ? $_POST['patLastName'] : '' ?>" placeholder="Patient Last Name"/>
+                        </div>
+                    </div>
+                    <div class="col">
+                    
+                    </div>
+                    <div class="col">
+                    
+                    </div>
+                </div>   
+                <div class="row">
+                    <div class="col">
+                        <button type="submit" name="SubmitButton" class="btn btn-primary">Submit</button>
+                    </div>
+                    <div class="col-10">
+                    
+                    </div>
+                </div>            
+                
+            </div> 
+        </form>
+
+
+        <?php                  
+            $datas = null;
```

Comment:
Should probably set this to an empty array ([]). Otherwise your foreach throws warnings.

Reply:
done, changed slightly and tested.